cpp/include/cugraph/dendrogram.hpp

```diff
@@ -27,10 +27,11 @@ class Dendrogram {
  public:
   void add_level(vertex_t first_index,
                  vertex_t num_verts,
-                 cudaStream_t stream,
+                 rmm::cuda_stream_view const &stream_view,
```

Comment:
Should we pass a const reference here?

Considering that
1) cuda_stream_view is just a wrapper for cudaStream_t.
https://github.com/rapidsai/rmm/blob/branch-21.08/include/rmm/cuda_stream_view.hpp#L108
2) and cuDF passes cuda_stream_view by value
https://github.com/rapidsai/cudf/blob/branch-21.08/cpp/include/cudf/detail/copy.hpp#L76
https://github.com/rapidsai/cudf/blob/branch-21.08/cpp/include/cudf/detail/groupby.hpp#L47

Reply:
Pushed the change to pass by value.